mitiq/pec/pec.py

```diff
@@ -120,10 +121,10 @@ def execute_with_pec(
 
     sampled_circuits = []
     signs = []
-
+    converted_circut, _ = convert_to_mitiq(circuit)
     for _ in range(num_samples):
         sampled_circuit, sign, _ = sample_circuit(
-            circuit, representations, random_state
+            converted_circut, representations, random_state
```

Comment:
Just a nit comment here that `converted_circut` should be `converted_circuit` :sweat_smile:

Nice find on this repeated conversion!